Apps/Immunization/src/Models/ImmunizationEvent.cs

```diff
@@ -75,23 +75,23 @@ public static ImmunizationEvent FromPHSAModel(ImmunizationResponse model)
                 ProviderOrClinic = model.ProviderOrClinic,
                 Status = model.Status,
                 Immunization = new ImmunizationDefinition() { Name = model.Name, ImmunizationAgents = ImmunizationAgent.FromPHSAModelList(model.ImmunizationAgents) },
-                Forecast = new ImmunizationForecast() // TODO: Needs to be mapped from the response
+                Forecast = model.ImmunizationForecast == null ? null : ImmunizationForecast.FromPHSAModel(model.ImmunizationForecast),
```

Comment:
Should we check and centralize the null check in the FromPHSAModel instead of doing it here?

Reply:
I think here is better, that way we dont even need to call the translation function

Reply:
not sure what is the translation function though. But it is fine for me.